pages/en/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os.md

```diff
@@ -0,0 +1,61 @@
+metadesc: Learn about the latest improvements for Android mobile apps on Chrome
+tags:
+  - product news
+  - android
+authors:
+  - fahdi
+featured:
+  title: A seamless mobile app experience
+  desc: Learn about the latest improvements for Android mobile apps on Chrome
+  images:
+    - image: /images/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os/hero.png
+      alt: An app shown on phone, tablet, and Chromebook
+hero:
+  image: /images/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os/hero.png
+  alt: An app shown on phone, tablet, and Chromebook
+  position: center
+date: 2021-09-10
+---
+
+In the Google Play Store, Chromebook users can find Android apps to do just about anything: get lost in a medieval role-playing game, make a weekly to-do list, listen to their favorite podcasts, or stay connected with friends and family. Users can even download and save content onto their Chromebooks to use their apps offline whenever they want.
+
+But for the most part, Android’s rich app experiences are designed for smartphones with much smaller screens than Chrome OS devices. When people use their apps on a laptop or tablet, a number of screen compatibility issues can crop up: content gets cut off in split-screen mode, portrait-mode-only apps launch in fullscreen, and in some cases, desktop use can even cause the app to crash.
+
+That’s why Chrome OS is introducing improvements to ensure a great mobile app experience, no matter what device users are on. And the best part is developers don’t need to do anything — we’ll be rolling out these changes along with the Android 11 update over the next few months.
+
+## Improved window management for Android mobile apps
+
+Starting in Chrome OS 93, improved window compatibility for mobile apps is now available on Android 11-enabled Chromebooks. With this first in a series of upcoming updates, apps will run in a window locked to phone or tablet orientations.
+
+Users can easily [switch between a predefined phone and tablet view](https://support.google.com/chromebook/thread/124854535/introducing-chrome-os-93-to-the-stable-channel?hl=en) by using a simple window resize UI that appears when they hover over the top of the app screen. Users can also disable this window management feature to enable full resizability by clicking the “Resizable” option.
+
+![](/images/posts/give-your-users-a-seamless-mobile-app-experience-on-chrome-os/window-management.jpg)
+
+This feature applies to most apps available in the Play Store not currently optimized for desktop use. Games and Media apps that have fullscreen mode enabled to [deliver more immersive experiences](https://developer.android.com/training/system-ui/immersive) will continue to launch in fullscreen mode, but users can change the window view at any time.
+
+For more details and tips, check out our post about [optimizing your Android apps for desktops.](/{{locale.code}}/android)
```

Comment:
Is this the right link?

Reply:
Yeah, that's what is in the article draft. Thanks for checking!